packages/mongodb-log-writer/src/mongo-log-writer.ts

```diff
@@ -186,6 +212,13 @@ export class MongoLogWriter extends Writable {
     await new Promise((resolve) => this._target.write('', resolve));
   }
 
+  writeIfEnabled(chunk: Parameters<typeof this.write>[0]): void {
```

Comment:
I could also use our current override of `this._write` and add the conditional there, it just feels less in our control so this feels more predictable to me.